packages/ddp-rate-limiter/ddp-rate-limiter-tests.js

```diff
@@ -77,6 +77,68 @@ testAsyncMulti("ddp rate limiter - matchers get passed correct arguments", [
   }
 ]);
 
+testAsyncMulti("ddp rate limiter - callbacks get passed correct arguments", [
+  function (test, expect) {
+    _.extend(this, createTestUser(test, expect));
+  },
+  function (test, expect) {
+    var self = this;
+    Meteor.call("addRuleToDDPRateLimiter", expect(function(error, result) {
+     self.ruleId = result;
+    }));
+  },
+  function (test, expect) {
+    Meteor.call('dummyMethod', function() {});
```

Comment:
I think it's very unlikely in this environment but the next async step/test might (???) be executed before the DDP message was received and cause a failure.  I think the `async` test handler is smart enough to know that there are no actual `expect`ations here but explicitly showing that we `expect` nothing, it would make the test more clear.